README.md

```diff
@@ -211,6 +213,15 @@ Allows you to increase the rollout percentage of an app that was previously rele
 
 3. **Rollout Fraction** *(String, Required)* - The new user fraction to increase the rollout to, specified as a number between 0 and 1 (e.g. `0.5` == `50%` of users)
 
+### Google Play - Halt
+
+Allows you to halt an app that was previously released to the **Rollout** track, and includes the following options:
+
+![Increase task](images/halt-task.png)
+
+1. **JSON Key Path** *(File path)* or **Service Endpoint** - The credentials used to authenticate with Google Play. This can be acquired from the [Google Developer API console](https://console.developers.google.com/apis) and provided either directly to the task (via the `JSON Auth File` authentication method), or configured within a service endpoint that you reference from the task (via the `Service Endpoint` authentication method). Note that in order to use the JSON Auth File method, the JSON file you get from the developer console needs to be checked into your source repo.
+2. **Package Name** *(String, Required)* - The unique package identifier (e.g. com.foo.myapp) of the app you wish to halt for.
```

Comment:
Could you please add description for Track input also?

Reply:
Maybe we can update the README doc after we have a full discussion and confirm all modifications.

Reply:
Nit: please don't forget to update it also 😃

Reply:
I have completed the update of the document. 
P.S. I found out that baseREADM.md and makeReadme.sh is outdated.